apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,389 @@
+                          data-content='<pre style="background-color: black" class="text-white rounded p-3 mb-0">${question.sync_errors_ansified}</pre>'
+                        >
+                          <i class="fa fa-times text-danger" aria-hidden="true"></i>
+                        </button>
+                      `
+                    : question.sync_warnings
+                      ? html`
+                          <button
+                            class="btn btn-xs mr-1 js-sync-popover"
+                            data-toggle="popover"
+                            data-trigger="hover"
+                            data-container="body"
+                            data-html="true"
+                            data-title="Sync Warnings"
+                            data-content='<pre style="background-color: black" class="text-white rounded p-3 mb-0">${question.sync_warnings_ansified}</pre>'
+                          >
+                            <i
+                              class="fa fa-exclamation-triangle text-warning"
+                              aria-hidden="true"
+                            ></i>
+                          </button>
+                        `
+                      : ''}
+                  ${question.display_name}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/topic'); %>", {
+                    topic: question.topic,
+                  })}
+                </td>
+                <td>
+                  ${renderEjs(__filename, "<%- include('../partials/tags'); %>", {
+                    tags: question.tags,
+                  })}
+                </td>
+                <td>
+                  ${maxPoints({
+                    max_auto_points: question.max_auto_points,
+                    max_manual_points: question.max_manual_points,
+                    points_list: question.points_list,
+                    init_points: question.init_points,
+                  })}
+                </td>
+                <td>${question.max_manual_points || '—'}</td>
+                ${showAdvanceScorePercCol
+                  ? html`
+                      <td
+                        class="${question.assessment_question_advance_score_perc === 0
+                          ? 'text-muted'
+                          : ''}"
+                        data-testid="advance-score-perc"
+                      >
+                        ${question.assessment_question_advance_score_perc}%
+                      </td>
+                    `
+                  : ''}
+                <td>
+                  ${question.mean_question_score
+                    ? `${question.mean_question_score.toFixed(3)} %`
+                    : ''}
+                </td>
+                <td>
+                  ${question.avg_question_score_perc
+                    ? `${question.avg_question_score_perc.toFixed(3)} %`
+                    : ''}
+                </td>
```

Comment:
I actually ended up asking @mwest1066 about this, and specifically about the query that this data comes from:

https://github.com/PrairieLearn/PrairieLearn/blob/c0d897f65ad0435b6b2941ed77b3e81c0a84e991/apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.sql#L17-L27

> what's going on here? AFAICT we end up just averaging a single value, which itself is already an average? We end up displaying both `aq.mean_question_score` (as "Mean score") and this `avg(aq.mean_question_score)` (as "Question score") on the assessment questions table. Will these ever be two different numbers? I spot-checked a few of Mariana's assessments and they're always the same numbers.

Reply:
Good catch! I agree that these two numbers will always be identical (and always were identical) and the "Question score"/`avg(aq.mean_question_score)` column can thus be deleted.

For background, this SQL CTE was introduced in commit https://github.com/PrairieLearn/PrairieLearn/commit/ace89b017165d19c31c0bc5e18555cd350777bd7#diff-efe7611442ecb931f08256216ddec98e6630ff0820eeff0dc4c508c19ac8d8dd as part of #891, authored by Paras Sud (undergrad doing a senior thesis with Craig) and committed by me (clearly with insufficient code review 😭).

Even when this code was written, we had the constraint: https://github.com/PrairieLearn/PrairieLearn/blob/57416712b135db16b0669665b15ce3853791d543/database/tables/assessment_questions.pg#L45 on `assessment_questions`, so the `SELECT` in the [quoted code above](https://github.com/PrairieLearn/PrairieLearn/pull/9680#discussion_r1559828767) can only ever find a single `assessment_questions` row for each `question_id`, making the `GROUP BY` and `avg()` completely pointless. That is, `avg(aq.mean_question_score)` will always be identical to `aq.question_score` and there is hence no point in having this CTE or the "Question score" column at all.

Reply:
Thanks for the careful spelunking! @mylesw27 can you go ahead and remove this CTE and the corresponding column from the HTML table?

Reply:
@mwest1066 Thanks for the explanation!

@nwalters512 The CTE and "Question Score" column have been removed in [9c796a1](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/9c796a178e71f3f826a791d721b88d09a8ed9baa).